projects/plugins/jetpack/extensions/blocks/math/block.json

```diff
@@ -0,0 +1,49 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 3,
+	"name": "jetpack/math",
+	"version": "0.1.0",
+	"title": "Math",
+	"keywords": [ "math", "formula", "equation", "mathml", "latex" ],
+	"category": "formatting",
+	"description": "Create and display math formulas with live preview and easy editing.",
+	"example": {
+		"attributes": {
+			"source": "\\frac{a}{b}"
+		}
+	},
+	"attributes": {
+		"source": {
+			"type": "string",
+			"default": ""
+		}
+	},
+	"supports": {
+		"html": true,
+		"color": {
+			"gradients": true
+		},
+		"spacing": {
+			"margin": true,
+			"padding": true
+		},
+		"typography": {
+			"fontSize": true
+		},
+		"align": [ "wide", "full" ],
+		"__experimentalBorder": {
+			"color": true,
+			"radius": true,
+			"style": true,
+			"width": true,
+			"__experimentalDefaultControls": {
+				"color": true,
+				"radius": true,
+				"style": true,
+				"width": true
+			}
+		}
+	},
```

Comment:
@alshakero it might be easier to split this PR into two: MathML and styles separately, as both have separate concerns to work through, so it would be faster keep momentum going in smaller PRs.